dist/pythonlibs/riotctrl_shell/requirements.txt

```diff
@@ -1 +1 @@
-git+ssh://git@github.com/RIOT-OS/riotctrl
+git+https://github.com/RIOT-OS/riotctrl
```

Comment:
This was mainly because at the start `riotctrl` was still private, but now that it isn't `https` is also possible: question is though: why? Is there a problem with using SSH?

Reply:
[It doesn't work without prior setup when using SSH](https://github.com/aabadie/RIOT/runs/836950372?check_suite_focus=true).

Reply:
Ahh. Makes sense!